guide/how-it-works/human-readable-addresses.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: guide
+title: Human readable addresses
+description: Easy-to-use bitcoin addresses via DNS payment instructions and the lightning address protocol.
+nav_order: 1
+parent: Payment request formats
+grand_parent: How it works
+permalink: /guide/how-it-works/human-readable-addresses/
+main_classes: -no-top-padding
+image: /assets/images/guide/how-it-works/human-readable-addresses/human-readable-addresses-preview.jpg
+
+---
+
+<!--
+
+Editor's notes:
+
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/human-readable-addresses/human-readable-addresses.jpg"
+   retina = "/assets/images/guide/how-it-works/human-readable-addresses/human-readable-addresses@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/human-readable-addresses/human-readable-addresses-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/human-readable-addresses/human-readable-addresses-mobile@2x.jpg"
+   alt-text = "The word bitcoin address written in binary and plain text"
+   width = 1600
+   height = 400
+   layout = "full-width"
+%}
+
+# Human readable addresses
+
+
+[Bitcoin addresses]({{'/guide/glossary/address/' | relative_url}}) and [payment requests]({{'/guide/how-it-works/payment-request-formats/' | relative_url}}) are long and cryptic, practically impossible to read, memorize, pronounce, understand, or type. We are used to being able to communicate our social media handles, or email addresses easily. That expectation has also extended to transacting value, with neo-banking and payment applications. There have been multiple attempts at making human-readable addresses a reality in bitcoin.
+
+The benefits to the user experience include:
+
+- Familiarity: A format that feels more human and familiar to users
+- Ease of sharing: A format that is easier to remember and recall and share in a social setting
+
+Note that human-readable addresses should be <b>only</b> used for direct communication. Whenever payment information is communicated digitally (copy & paste, links, QR codes, messaging, etc), regular addresses should be used for best privacy and security.
+
+On this page, we focus primarily on the [DNS Payment Instructions proposal](https://github.com/bitcoin/bips/pull/1551), with some information about [Lightning Addresses](https://lightningaddress.com). They pursue similar goals, but in very different ways.
```

Comment:
We have promoted things which do not have wide adoption as yet like [Sign in with bitcoin](https://bitcoin.design/guide/how-it-works/sign-in-with-bitcoin) but here it's spoken about in the abstract instead of strong recommendation to WebLN over LNURL-Auth for example.

From my recollection this is the first time a payment protocol will be added to the guide at such an early stage, against something else which has already had adoption that it's to supplant.

Stephen created a separate campaign page for Unified QR Codes to help drive its' adoption. Will this page be the primary campaign page for the BIP?

Asking because we have been quite reserved so far, so wondering if it's a shift in our editorial policy for BDG.

Reply:
Good point. I'll add that this is an early proposal.

Overall, I think we can be more forward-looking, especially if it could maybe help the ecosystem figure out how to move forward. We just need the proper disclaimers.

Reply:
Please note that all replies here are hinting that the forward-looking aspects are the most questionable/doubtful.

Reply:
Also see the [Stabilizing bitcoin value](https://bitcoin.design/guide/how-it-works/stabilizing-bitcoin-value/) which has a disclaimer about the techniques being experimental and such.

> Please note that all replies here are hinting that the forward-looking aspects are the most questionable/doubtful.

Isn't it normal and to be expected that the forward-looking aspects would be the ones that are discussed the most?

> Will this page be the primary campaign page for the BIP?

I do not know how this page will be used. There are no plans for a campaign. Matt made a proposal, this page is about how this proposal could surface in UIs.

Reply:
@johnsBeharry I added a note in the page about that proposal being brand new and still being discussed.